### PR TITLE
showItemInbag feature done + fixed aria-label on button

### PR DESCRIPTION
**Pull request by ItsMatDev**

### CODE REVIEW

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 1]
```diff
@@ -57,3 +57,22 @@
   border: 2px solid $primaryColor;
   color: $primaryColor;
 }
+.itemsContainer {
+  display: flex;
+  align-items: center;
+  margin-top: 5vh;
+  margin-left: 5vh;
```
Comment (pagouillart) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: vw

**Review by pagouillart — APPROVED** [review]

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 2]
```diff
@@ -23,18 +25,42 @@ function Inventory() {
...
+        >
+          X
+        </button>
+        <main className="itemsContainer">
+          {bag.find((e) => e === "egg") ? (
+            <img
+              src={egg}
+              className="eggImgInBag"
+              alt="L'oeuf"
+              onMouseEnter={() =>
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: what about keyboard navigation?

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 3]
```diff
@@ -23,18 +25,42 @@ function Inventory() {
...
+                setInfo(
+                  <article className="eggPopUpInBag">
+                    L'oeuf du quetzalcoatlus
+                  </article>
+                )
+              }
+              onMouseLeave={() => setInfo("")}
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: what about keyboard navigation?

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss (hunk 1 above)
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: No vh in width

Comment (ItsMatDev) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: done

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 3 above)
Comment (ItsMatDev) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: done

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 2 above)
Comment (ItsMatDev) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx]: done

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss (hunk 1 above)
Comment (ItsMatDev) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**